src/formattedcode/output_spdx.py

```diff
@@ -344,7 +344,7 @@ def write_spdx(output_file, files, tool_name, tool_version, notice, input_file,
     # encode to UTF8 bytes.
 
     if package.files:
-        from StringIO import StringIO
+        from io import StringIO
```

Comment:
Again repetition of commit https://github.com/nexB/scancode-toolkit/pull/1534/commits/1e95b2c025acc82aec3e673012a04ee96f808602